doc/topics/the-event-loop-timers-and-nexttick.md

```diff
@@ -131,78 +131,78 @@ someAsyncOperation(function () {
 });
 ```
 
-When the event loop enters the `poll` phase, it has an empty queue
+When the Event Loop enters the `poll` phase, it has an empty queue
 (`fs.readFile()` has not completed) so it will wait for the number of ms
 remaining until the soonest timer's threshold is reached. While it is
 waiting 95 ms pass, `fs.readFile()` finishes reading the file and its
 callback which takes 10 ms to complete is added to the `poll` queue and
 executed. When the callback finishes, there are no more callbacks in the
-queue, so the event loop will see that the threshold of the soonest
+queue, so the Event Loop will see that the threshold of the soonest
 timer has been reached then wrap back to the `timers` phase to execute
 the timer's callback. In this example, you will see that the total delay
-between the timer  being scheduled and its callback being executed will
+between the timer being scheduled and its callback being executed will
 be 105ms.
 
-Note: To prevent the `poll` phase from starving the event loop, libuv
+Note: To prevent the `poll` phase from starving the Event Loop, libuv
 also has a hard maximum (system dependent) before it stops `poll`ing for
 more events.
 
-### I/O callbacks:
+### `I/O callbacks`:
 
-This phase executes callbacks for some system operations such as types 
+This phase executes callbacks for some system operations such as types
 of TCP errors. For example if a TCP socket receives `ECONNREFUSED` when
 attempting to connect, some \*nix systems want to wait to report the
 error. This will be queued to execute in the `I/O callbacks` phase.
 
-### poll: 
+### `poll`:
```

Comment:
Ah, one more thing, these event loop phases are back-ticked all over the document. I think it really helps for them to stand out from normal text, but I agree that the reader should probably not think of them as code. What do you think about a different designation for these phases, maybe italics? or just quotes?